src/main/java/com/michalszekalski/bootcamp_zad25/TaskController.java

```diff
@@ -0,0 +1,57 @@
+package com.michalszekalski.bootcamp_zad25;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+public class TaskController {
+
+    private TaskRepository taskRepository;
+
+    public TaskController(TaskRepository taskRepository) {
+        this.taskRepository = taskRepository;
+    }
+
+    @GetMapping("/")
+    public String home(@RequestParam(required = false) Boolean isDone, Model model) {
+        List<Task> tasks;
+
+        if (isDone == null) {
+            tasks = taskRepository.findAllOrderByDeadlineAsc();
+        } else {
+            tasks = taskRepository.findByIsDoneOrderByDeadlineAsc(isDone);
+        }
+
+        model.addAttribute("filterMode", isDone);
+        model.addAttribute("allTasks", tasks);
+        model.addAttribute("newTask", new Task());
+        model.addAttribute("localDateNow", LocalDate.now());
+        return "home";
+    }
+
+    @PostMapping("/addTask")
+    public String addTask(Task task) {
+        task.setIsDone(false);
```

Comment:
Tutaj też nie musisz tego poprawiać, ale jeżeli zadania domyślnie mają być ustawione jako "niewykonane", to albo zmieniłbym typ Boolean na typ prosty boolean (na poziomie bazy danych możnaby dodać wtedy false jako wartość domyślną). Ten stan pośredni (null) czasami się przydaje, tutaj mógłby wyrażać np. stan "aktualnie wykonywane", ale w Twoim przypadku tego nei wykorzystujesz, a łatwo zapomnieć o tym, że gdzieś trzeba ustawić to pole ręcznie na false